main.go

```diff
@@ -607,11 +607,15 @@ func deployCloudComponents(plan types.Plan) error {
 	if plan.SCM == "gitlab" {
 		gitlabEnv = "GITLAB=true"
 	}
+	networkPoliciesEnv := ""
+	if plan.NetworkPolicies{
```

Comment:
I think `gofmt` was not executed here @martindekov

Reply:
you are right @matipan for some reason VS Code didnt trigger that automatically on save, not sure why, thanks !